main_AdamTadele.py

```diff
@@ -0,0 +1,48 @@
+import random
+
+names = []
+
+
+def add_name():
+    print('Press x stop entering a name')
+    print('Enter a name:')
+    while True:
+        name = input()
+
+        if name == 'x':
+            # Check at least two names have been entered
+            if len(names) < 2:
```

Comment:
Think about o the logic here. Thnk about this:

if len(names) >= 2:
     break
else:
      print('Please enter at least at least two names")

This makes more sense to me. What do you think?

Reply:
You can even use an AND here as well so that there isn't an if nested in an if.